testing/test_terminal.py

```diff
@@ -718,3 +719,63 @@ def pytest_terminal_summary(terminalreporter):
         *==== hello ====*
         world
     """)
+
+@pytest.mark.parametrize("exp_color, exp_line, stats_arg", [
+    # The method under test only cares about the length of each
+    # dict value, not the actual contents, so tuples of anything
+    # suffice
+
+    # Important statuses -- the highest priority of these always wins
+    ("red",    "1 failed",               {"failed":     (1,)}),
```

Comment:
Well, because, as PEP8 itself says:

>  Some other good reasons to ignore a particular guideline:
>
>  1.  When applying the guideline would make the code less readable, even for someone who is used to reading code that follows this PEP. 

IMO, code that's essentially a big table is more readable when it's formatted as such.

I wasn't sure what pytest's conventions are, so I submitted the code in the format I prefer.  If that isn't what pytest prefers, I'm happy to change it -- but I'd rather not do that until I'm finished with it, i.e. until everything else about the PR has been approved.